### PR TITLE
{compiler}[GCCcore/8.3.0] LLVM v9.0.1

### PR DESCRIPTION
**Pull request by Falkor**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-002 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/fe1c0bd7a92bbfe99f5346706025d1ba for a full test report.

**Comment by Falkor** [comment]
again the checks fail way before arriving to the real tests runs ("install OS & Python packages" fail). Is there a way to force the unit tests to restart?

File: easybuild/easyconfigs/l/LLVM/LLVM-9.0.1-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+dependencies = [
+    ('ncurses', '6.1'),
+    ('zlib', '1.2.11'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON '
+# required to install extra tools in bin/
+configopts += '-DLLVM_INSTALL_UTILS=ON -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_ZLIB=ON -DLLVM_ENABLE_RTTI=ON '
+configopts += '-DCMAKE_BUILD_TYPE=Release '
```
Comment (lexming) [inline easybuild/easyconfigs/l/LLVM/LLVM-9.0.1-GCCcore-8.3.0.eb:36]: Recently we added a specific option to automatically handle `BUILD_TYPE`
```suggestion
build_type = 'Release'
```

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-101 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/9c61d87beb1d0c6f353ecdec55e60b90 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node376.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/5ac2912a21fb1404e5ea6e016b6cf564 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @Falkor !

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/6e0e41db063cfc633a73ec613fddf264 for a full test report.

**Pull request merged**